libsrc/Makefile

```diff
@@ -91,7 +93,7 @@ define INSTALL_recipe
 
 $(if $(prefix),,$(error variable `prefix' must be set))
```

Comment:
BTW - didn' t notice it earlier - but isn't that kind of broken here? I mean it requires $prefix but then uses $DESTDIR, which it doesn't require?